.github/PULL_REQUEST_TEMPLATE/pull_request_template.md

```diff
@@ -0,0 +1,75 @@
+<!--  All Submissions -->
+
+
+## Community Note
+<!-- Please leave the community note as is. -->
+* Please vote on this PR by adding a :thumbsup: [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original PR to help the community and maintainers prioritize for review
+* Please do not leave "+1" or "me too" comments, they generate extra noise for PR followers and do not help prioritize for review
+
+
+## PR Checklist
+
+- [ ] I have followed the guidelines in our [Contributing Documentation](../contributing/README.md).
```

Comment:
I'm not quite sure this (and other) relative link will work for a PR, as the PR link is like:

`https://github.com/hashicorp/terraform-provider-azurerm/pull/25103`, which will be `https://github.com/hashicorp/terraform-provider-azurerm/pull/contributing/README.md` after evaluated.

Also it's worth mentioning that there is one special guideline in the `contributing` folder is about how to create a PR: https://github.com/hashicorp/terraform-provider-azurerm/blob/main/contributing/topics/guide-opening-a-pr.md. When users check this checkbox, it means they have followed that guideline also, while that guideline by its name shall contain everything in this template, which means we only need to put things there and only leave one checkbox here, which likely make PR authors life much easier and avoid inconsistency between this template and that guideline?

Reply:
@magodo Thanks for catching the relative links, I have updated them so they should work now.

As for the formatting suggestions, we are discussing ways to reduce checkbox fatigue per your suggestions. I am not sure what changes we will make yet, but please continue to give us feedback once the PR template is live, since it is a living document and we may need to make future changes depending on user experience.